Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -239,10 +249,23 @@ def retrieve_and_display_info(formula_or_cask, name, repositories, args:, ambigu
 
     ohai title
     puts <<~EOS
-      Current #{formula_or_cask.is_a?(Formula) ? "formula version:" : "cask version:   "}  #{current_version}
+      Current #{version_name}:  #{current_version}
       Latest livecheck version: #{livecheck_latest}
       Latest Repology version:  #{repology_latest}
+      Open pull requests:       #{pull_requests || "none"}
     EOS
-    puts "Open pull requests:       #{pull_requests}" unless pull_requests.nil?
+
+    return unless args.open_pr?
+    return if pull_requests.present?
+
+    new_version = if livecheck_latest != current_version
```

Comment:
```suggestion
    new_version = if livecheck_latest > current_version
```

There are rare instances where a formula version is newer than the livecheck (or repology) version. Sometimes this is intentional (e.g., temporarily switching to an unstable version for technical reasons) and sometimes it's just because we need to add/modify a `livecheck` block to identify the correct latest version again (e.g., due to an upstream change).

Unless I'm overlooking something, only checking for inequality would lead to an unwanted PR for an older version in this scenario.

Reply:
Do we have unit tests for bump command? I think we can also cover the said scenarios over there?

Reply:
Nope, we don't have unit tests and we don't tend to unit test commands,